packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -121,12 +123,18 @@ class AmplifyAuthCognitoPluginTest {
     }
 
     @Test
-    fun confirmSignUp_returnsSuccess() {
+    fun confirmSignUpWithoutOptions_returnsSuccess() {
         // Arrange
         doAnswer { invocation: InvocationOnMock ->
             plugin.prepareSignUpResult(mockResult, mockSignUpResult)
             null as Void?
-        }.`when`(mockAuth).confirmSignUp(anyString(), anyString(), ArgumentMatchers.any<Consumer<AuthSignUpResult>>(), ArgumentMatchers.any<Consumer<AuthException>>())
+        }.`when`(mockAuth).confirmSignUp(
+            anyString(),
```

Comment:
Just to confirm - we are only making this change because under the hood amplify-android will always have a default value and/or empty map here, so we are changing the pattern matcher?

Reply:
amplify-android will invoke `AuthConfirmSignUpOptions.default()` to get an empty map for the `options` parameter if it's not supplied (if this is what you were confirming about :))